README.md

```diff
@@ -51,18 +52,28 @@ Choose the right package for your platform.
 (\*) ROCm support is an experimental feature. Refer to the [docs](https://docs.cupy.dev/en/latest/install.html#using-cupy-on-amd-gpu-experimental) for details.
 
 Append `--pre -f https://pip.cupy.dev/pre` options to install pre-releases (e.g., `pip install cupy-cuda11x --pre -f https://pip.cupy.dev/pre`).
-See the [Installation Guide](https://docs.cupy.dev/en/stable/install.html) if you are using Conda/Anaconda or building from source.
 
-## Run on Docker
+### Conda
+
+Binary packages are also available for Linux and Windows on [Conda-Forge](https://anaconda.org/conda-forge/cupy).
+
+| Platform              | Architecture                | Command                                                       |
+| --------------------- | --------------------------- | ------------------------------------------------------------- |
+| CUDA                  | x86_64 / aarch64 / ppc64le  | `conda install -c conda-forge cupy`                           |
+
+If you need to use a particular CUDA version (say 11.8), you can do `conda install -c conda-forge cupy cudatoolkit=11.8`.
```

Comment:
This is future proof (works now and will continue to work after CUDA 12 is up):
```suggestion
If you need to use a particular CUDA version (say 11.8), you can do `conda install -c conda-forge cupy cudatoolkit cuda-version=11.8`.
```
xref: [`cuda-version`](https://github.com/conda-forge/cuda-version-feedstock/blob/main/recipe/README.md)

Reply:
Ah it can be simplified
```suggestion
If you need to use a particular CUDA version (say 11.8), you can do `conda install -c conda-forge cupy cuda-version=11.8`.
```

Reply:
Thanks! Also updated the installation guide.